src/components/Header/HeaderContact.js

```diff
@@ -6,6 +6,7 @@ import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {COLORS, SIZES} from '../../utils/theme';
 import GlobalButton from '../Button/GlobalButton';
 import dimen from '../../utils/dimen';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a redundancy in the import of color utilities. You have imported `COLORS` from `'../../utils/theme'` and `colors` from `'../../utils/colors'`. If these two imports are serving the same purpose, consider using only one to maintain consistency and avoid confusion.
```diff
- import {COLORS, SIZES} from '../../utils/theme';
- import {colors} from '../../utils/colors';
+ import {COLORS, SIZES} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->